libvast/src/schema.cpp

```diff
@@ -239,9 +239,15 @@ get_schema_dirs(const caf::actor_system_config& cfg,
     else if (const char* home = std::getenv("HOME"))
       result.insert(path{home} / ".local" / "share" / "vast" / "schema");
   }
-  if (auto user_dirs
-      = caf::get_if<std::vector<std::string>>(&cfg, "vast.schema-paths"))
-    result.insert(user_dirs->begin(), user_dirs->end());
+  if (auto dirs = caf::get_if<std::vector<std::string>>( //
```

Comment:
I think it would be nice if we could factor all CLI deprecations into a single function so that we only have to check one place in future releases.

Reply:
Not sure if that's worth the effort tbh.

Reply:
To clarify, this would ideally be done when parsing the options, but we cannot use the logger at that point. So I'd like to keep it as is currently.

Reply:
Marking this as resolved as per Slack. `git grep deprecated` will suffice for now.